src/render_tree.rs

```diff
@@ -14,8 +18,8 @@ pub mod util;
 /// to draw an object
 #[derive(Debug, PartialEq)]
 pub struct Position {
-    pub x: f64,
-    pub y: f64,
+    pub x: f32,
+    pub y: f32,
```

Comment:
Using `f32` to correspond to C's `float` type

Reply:
`f64` should be C's `double`, so you can leave that as `f64`

Reply:
I can change the API to use `double` instead of `float` and revert back to `f64`